web/stories/EntityTableRow.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { EntityTableRow } from '../src/components';
+
+const entities = {
```

Comment:
3rd time these have shown up! time to move to a separate file, I think.

Reply:
Yeah. purposefully kept these declarations WET as they are duplicates of the test data for the legacy API (and the Dgraph backend data). I think leave the DRY implementation for `nx`?